### PR TITLE
fix cbf reader

### PR DESCRIPTION
**Pull request by jkotan**
It fixes a bug in cbf reader

### CODE REVIEW

File: test/reader_test/fio_reader_test.cpp [hunk 1]
```diff
@@ -21,7 +21,7 @@
 //      Author: Eugen Wintersberger <eugen.wintersberger@desy.de>
 //
 #include <boost/test/unit_test.hpp>
-#include <boost/test/floating_point_comparison.hpp>
+#include <boost/test/tools/floating_point_comparison.hpp>
```
Comment (yuelongyu) [inline test/reader_test/fio_reader_test.cpp:24]: with this change, i think it's better to change the boost requirements in installation: Boost (>=1.55). In old boost, this header file is in another path which previously was used.

**Review by yuelongyu — CHANGES_REQUESTED** [review]
change the boost requirements in README.md

File: test/reader_test/fio_reader_test.cpp (hunk 1 above)
Comment (jkotan) [reply test/reader_test/fio_reader_test.cpp:24]: Done

**Review by yuelongyu — APPROVED** [review]
looks good to me now. :+1:

**Pull request merged**